index.html

```diff
@@ -4189,7 +4189,22 @@ <h3>JSON-LD</h3>
         </p>
 
```

Comment:
(I cannot directly edit the text from github, copying here the problematic part:

> [[!JSON-LD11]] is a JSON-based format used to serialize <a href="https://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>. The syntax is designed to easily integrate into deployed systems already using JSON, and provides a smooth upgrade path from JSON to [[!JSON-LD11]].

The 2nd sentence looks a bit strange to me.  It says 

> JSON-LD 'provides a smooth upgrade path from JSON' to… JSON-LD.

I.e., it provides a smooth upgrade to itself... which is a bit circular. My proposal would be to remove the second part of the sentence, to say:

> [[!JSON-LD11]] is a JSON-based format used to serialize <a href="https://www.w3.org/TR/ld-glossary/#linked-data">Linked Data</a>. The syntax is designed to easily integrate into deployed systems already using JSON.

Reply:
Agreed. It does sound circular. I'll remove that last bit.